index.html

```diff
@@ -896,7 +898,14 @@
           [SameObject] readonly attribute PushSubscriptionOptions options;
           ArrayBuffer? getKey(PushEncryptionKeyName name);
           Promise&lt;boolean&gt; unsubscribe();
-          serializer;
+
+          PushSubscriptionJSON toJSON();
+        };
+
+        dictionary PushSubscriptionJSON {
+          USVString endpoint;
+          DOMTimeStamp? expirationTime;
+          record keys;
```

Comment:
I've left this as a comment for now and filed an issue on ReSpec:
https://github.com/w3c/respec/issues/1297